app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class CustomerItemsController < ApplicationController
+  def create
+    customer = Customer.find(params[:id])
+    item = Item.find(params[:item_id])
+    customer.items << item
```

Comment:
Yes, this works. Alternatively, you could have created a model method in Customer called `add_item(item)`, but that is slightly nit picky. :)